src/composables/useTresEventManager/index.ts

```diff
@@ -15,8 +16,8 @@ export interface TresEventManager {
    * So we need to track them separately
    * Note: These are used in nodeOps
    */
-  registerPointerMissedObject: (object: TresObject) => void
-  deregisterPointerMissedObject: (object: TresObject) => void
+  registerPointerMissedObject: (object: any) => void
```

Comment:
Please don't use `any`. child coming from the nodeops is always gonna be `Object3D<Object3DEventMap>`

Reply:
Hey @alvarosabu , I changed `any` to `unknown` here. (Will push in a moment.)

Just to clear up this part because I don't want to merge if you're not on board with the reasoning:

> child coming from the nodeops is always gonna be Object3D<Object3DEventMap>

That was true prior to this PR.

But with this PR, the narrowing/typing is pushed down into `registerPointerMissed`.

Now `registerPointerMissed` will receive all `child: TresObject` from `insert(child: TresObject, parent: TresObject)` – not just `Object3D`. It's up to `registerPointerMissed` to decide if the passed `TresObject` requires registration. 

This is intended to make the function more robust and simplify code in the caller (`nodeOps`).